charts/springboot/templates/service.yaml

```diff
@@ -8,6 +8,10 @@ metadata:
     release: {{ .Release.Name }}
     heritage: {{ .Release.Service }}
     {{ required "'project' value not found" .Values.project }}-app: "true"
+  {{- if .Values.service.annotations }}
+  annotations:
+{{ toYaml .Values.service.annotations | indent 6 }}
```

Comment:
Consider using `nindent 6` and indent the `{{ toYaml ...}` for better readability.

Reply:
Cool, nice tip. Fixed. Also `4` spaces were enough apparently